src/NetMQ/Core/Mechanisms/CurveMechanismBase.cs

```diff
@@ -45,7 +45,7 @@ public override void Dispose()
         {
             Assumes.NotNull(m_box);
```

Comment:
Should remove this line too.

Reply:
I agree, line has been removed and I am creating a new pull request.